### PR TITLE
Main Menu Mod Version

### LINKED ISSUE #1335: Mod version on main menu
**Describe your suggestion in full detail below:**
Using `interface` folder magic, mention the current mod version on the main menu.

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Developer changelog:
- Added mod version to the main menu
- Close #1335

#### How to test:
- You can change version text in `000_wc_mod_version.csv`.

### CODE REVIEW

**Comment by MemeWizard** [comment]
@zumbak04 update to 3.3.0 when

**Comment by zumbak04** [comment]
> @zumbak04 update to 3.3.0 when

I wouldn't copy vanilla files anyway, I believe it may be done by overwrites in separate files.

**Comment by MemeWizard** [comment]
> I believe it may be done by overwrites in separate files.

Well. I'm interested how you will do it.

File: localisation/000_wc_mod_version.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+wc_mod_version;Early Version;Early Version;Early Version;;Early Version;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/000_wc_mod_version.csv:2]:
```suggestion
wc_mod_version;Early 1.8.0;Early 1.8.0;Early 1.8.0;;Early 1.8.0;;;;;;;;;x
```

**Pull request merged**